service-impl/audit-service-impl/logger-audit-service/src/main/java/uk/gov/gchq/palisade/audit/service/impl/LoggerAuditService.java

```diff
@@ -37,20 +44,73 @@
  * </pre>
  */
 public class LoggerAuditService implements AuditService {
+    private interface Handler {
+        void handle(Object o);
+    }
+
     private static final Logger LOGGER = LoggerFactory.getLogger(LoggerAuditService.class);
+    private static final Map<Class, Handler> DISPATCH = new HashMap<Class, Handler>();
+
+    //translate class object to handler
+    static {
+        //handler for ExceptionAuditRequest
+        DISPATCH.put(ExceptionAuditRequest.class, (o) -> {
+            requireNonNull(o, "exceptionAuditRequest");
+            AuditRequestWithContext auditRequestWithContext = (AuditRequestWithContext) o;
+            ExceptionAuditRequest exceptionAuditRequest = (ExceptionAuditRequest) o;
```

Comment:
CAsting the same object twice is unnecessary here. Cast once to ExceptionAuditRequest subclass and the remove the other one.

Reply:
done.